### ISSUE TITLE
ERROR: Can't recognize collecitons/folders

### ISSUE THREAD

**Issue by guilh-n-l** [Rigellute/spotify-tui#1013]
![image](https://user-images.githubusercontent.com/110556149/185961956-aabc61e6-3a32-4922-ac30-7c504cb9564d.png)

I tried deleting all my folders to try to fix it but the errror is still there. Also logged out and in to see if it worked but to no avail.

**Comment by guilh-n-l** [comment]
I use windows 10 btw

**Comment by AntoineGagne** [comment]
I might be wrong (I am not really familiar with the codebase), but I think this is an issue with [`rspotify`](https://github.com/ramsayleung/rspotify) which is one of the dependencies of the project. `spotify-tui` seems to be using version `0.10.0` which do not seem to support this enum variant. 

The [`master`](https://github.com/ramsayleung/rspotify/blob/2aca7c67ff94da3ffc3fb346d93b7c9ebc911cc3/rspotify-model/src/enums/types.rs#L41) branch of `rspotify` seems to support them though, but trying to upgrade the dependency to its latest version seems to introduce some issues with the application.

In the meantime, as a quick patch, I forked `rspotify` and created a branch where I added the enum variant. Locally, I can then modify the `Cargo.toml` to point to it and recompile it:

```toml
# ...
[dependencies]
rspotify = { git = "https://github.com/AntoineGagne/rspotify.git", branch = "feature/add-collection" }
# ...
```

Sorry if this is not that helpful.

EDIT: It seems to also be a duplicate of #912.

**Comment by sewnie** [comment]
@AntoineGagne i don't believe its that easy to fix, latest versions of rspotify break spotify-tui.

**Comment by AntoineGagne** [comment]
I am definitely not suggesting it is easy to fix. From what I can see, there has been pretty big changes between the version of `rspotify` used here and the latest one. My patch is only a bandaid and it just adds the missing type to the variant that is complaining in the error message. It is not a viable long-term solution (patching the downstream dependency that is).
